lib/ClangImporter/ImportDecl.cpp

```diff
@@ -2410,7 +2425,13 @@ namespace {
                                 (!cxxRecordDecl->hasDefaultConstructor() ||
                                  cxxRecordDecl->ctors().empty());
       }
-      if (hasZeroInitializableStorage && needsEmptyInitializer) {
+
+      // TODO: builtin "zeroInitializer" does not work with non-escapable
+      // types yet. Don't generate an initializer.
```

Comment:
Do you know if we're tracking this issue yet?

Reply:
I wasn't able to find anything. Will ask around if people are aware of this.

Reply:
I opened one just in case: rdar://143828851